sparse_ho/models/svm.py

```diff
@@ -339,14 +342,20 @@ def get_hessian(self, X, y, mask, dense, log_C):
         """
         C = np.exp(log_C)
         full_supp = np.logical_and(self.dual_var != 0, self.dual_var != C)
-        if issparse(X):
-            Xy = X[full_supp, :].multiply(y[full_supp, None])
-            return Xy @ Xy.T
-        else:
-            Xy = (y[full_supp] * X[full_supp, :].T)
-            return Xy.T @ Xy
 
-    def _get_jac_t_v(self, X, y, jac, mask, dense, C, v, n_samples):
+        X_m = X[full_supp, :]
+        y_m = y[full_supp]
+        size_supp = X_m.shape[0]
+
+        def mv(v):
+            if issparse(X):
```

Comment:
those two lines in the if else are the same

Reply:
done